lib/payload-persister.js

```diff
@@ -14,75 +14,82 @@ var CACHE = LRU({
   }
 });
 
-// TODO prioritize
-// TODO add a function that determines whether a given persister is appropriate
-var persisters = fs.readdirSync(path.join(__dirname, "persisters"))
-  .map(function(x) {
-    var persister = require(path.join(__dirname, "persisters", x));
-
-    return [persister.protocol, persister];
-  })
-  .reduce(function(obj, data) {
-    obj[data[0]] = data[1];
-
-    return obj;
-  }, {});
-
-module.exports = {
-  load: function(payload, callback) {
-    var uri = payload.uri;
-
-    if (uri && typeof uri === "string") {
-      uri = url.parse(uri);
-    }
+module.exports = (function() {
```

Comment:
When executing things in a `sync` context, it felt weird to require that I have a `AWS_DYNAMODB_TABLE` environment var or else it would error out. When we initialize the persisters, that triggers the loading of the dynamo db persister and requires that var, even though the sync decider doesn't use any persisters (because both the sync and non-sync workers are required by `decider.js`). The use case here would be, someone trying to run the local tiler on their machien, but being blocked because of a dynamo DB thing. Packing the persister initialization up here means that you'll only get hit with that call if you're trying to use the non-sync decider context.